### PR TITLE
Fix @types/livereload typedef

### PR DESCRIPTION
**Pull request by fxdave**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
The definition was wrong: 
    The type `WebSocket` was missing from the imports.

### CODE REVIEW

**Comment by MysteryBlokHed** [comment]
The imports in the types don't need to match the imports in the library, unless the import is actually used in the type definitions

**Comment by fxdave** [comment]
@MysteryBlokHed The imports are used in the type definition, see the whole file not just the changes. It wasn't imported before so I don't know how did it pass the tests, but tsc showed the error that I had to fix.

**Comment by MysteryBlokHed** [comment]
Ah, I see it now. I must've missed it at first.

File: types/livereload/index.d.ts [hunk 1]
```diff
@@ -3,7 +3,7 @@
 // Definitions by: Hector Osuna <https://github.com/FanGoH/>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
-import { Server as WebSocketServer } from 'ws';
+import { Server as WebSocketServer, WebSocket } from 'ws';
```
Comment (amcasey) [inline types/livereload/index.d.ts:6]: Sorry, I'm also confused about what problem this fixes.  The tests pass for me after adding the export below.

File: types/livereload/index.d.ts [hunk 2]
```diff
@@ -44,7 +44,7 @@ interface CreateServerConfig extends ServerConfig {
 }
 
 /** Live Reload Server object, provides main functionality */
-declare class LiveReloadServer extends EventEmitter {
+export class LiveReloadServer extends EventEmitter {
     config: ServerConfig;
```
Comment (amcasey) [inline types/livereload/index.d.ts:48]: Seems like you might as well export `ServerConfig` etc as well.

**Review by amcasey — CHANGES_REQUESTED** [review]

File: types/livereload/index.d.ts (hunk 1 above)
Comment (fxdave) [reply types/livereload/index.d.ts:6]: The test is falsepositive then. When I used this package, I got WebSocket type was undefined error.

File: types/livereload/index.d.ts (hunk 2 above)
Comment (fxdave) [reply types/livereload/index.d.ts:48]: The package is not mine but you are right about this. I will fix that.

Comment (fxdave) [reply types/livereload/index.d.ts:48]: Fixed

**Review by amcasey — APPROVED** [review]
Thanks!

**Comment by fxdave** [comment]
Ready to merge

**Pull request merged**